quickwit/quickwit-cli/src/index.rs

```diff
@@ -501,6 +536,31 @@ pub async fn create_index_cli(args: CreateIndexArgs) -> anyhow::Result<()> {
     Ok(())
 }
 
+pub async fn update_index_cli(args: UpdateIndexArgs) -> anyhow::Result<()> {
+    debug!(args=?args, "create-index");
+    println!("❯ Updating index...");
+    let storage_resolver = StorageResolver::unconfigured();
+    let file_content = load_file(&storage_resolver, &args.index_config_uri).await?;
+    let index_config_str: String = std::str::from_utf8(&file_content)
```

Comment:
Use `String::from_utf8` directly.

Reply:
`String::from_utf8` takes an owned `Vec<u8>` and there isn't a simple way to get that from `OnwedBytes`.